tests/common.go

```diff
@@ -594,8 +596,22 @@ func ValidateVolumes(ctx *scheduler.Context, errChan ...*chan error) {
 	context("For validation of an app's volumes", func() {
 		var err error
 		Step(fmt.Sprintf("inspect %s app's volumes", ctx.App.Key), func() {
-			appScaleFactor := time.Duration(Inst().GlobalScaleFactor)
-			err = Inst().S.ValidateVolumes(ctx, appScaleFactor*defaultVolScaleTimeout, defaultRetryInterval, nil)
+			vols, err := Inst().S.GetVolumes(ctx)
+			if err != nil {
+				logrus.Errorf("Failed to get app %s's volumes", ctx.App.Key)
+				processError(err, errChan...)
+			}
+			volScaleFactor := 1
+			if len(vols) > 10 {
+				// Take into account the number of volumes in the app. More volumes will
+				// take longer to format if the backend storage has limited bandwidth. Even if the
+				// GlobalScaleFactor is 1, high number of volumes in a single app instance
+				// may slow things down.
+				volScaleFactor = len(vols) / 10
```

Comment:
I think golang are treating them as integer by default. Is that what is expected? (e.g. only integer scale factor)
or we are expecting floating scale factor?

Reply:
Yes, just integer.